server/reportConfiguration/constants/typeHints.ts

```diff
@@ -14,3 +16,49 @@ export const typeHints: Partial<Record<Type, string>> = {
     'Includes suspected and reported self-harm. Do not use to report a noose, unless it’s around the neck or applying pressure.',
   TOOL_LOSS_1: 'Do not use for radio and key or lock compromises. They are separate incident types.',
 }
+
+/**
+ * Short titles displayed when generating titles in the form “About the incident”, eg. “About the assault”.
+ * A *family* of types shares one title.
+ */
+export const shortTypeTitles: Partial<Record<TypeFamily, string>> = {
```

Comment:
Isn't this the lower-cased family description on the API side? Any reason to hardcode those? Maybe there is some special exception here I'm missing?

Reply:
they weren't the same initially in the content spreadsheet i took this from. i decided to still keep this map in case the words diverge again. it's a bit more explicit even if it's duplication of text 🤷

Reply:
Why would they need to differ tho? And on the API side (in the title generation) the rule is simpler so I'd argue it would be consistent to do the same here.

But that's just my 2 cents.

Reply:
i agree, they don't _need_ to be different. but even now, not all match! check this one out:
family code: `MOBILE_PHONE`
type & family description: “Mobile phone”
question page title: “About the mobile phone incident” (surprise!)

Reply:
OK, so **there are** differences. In this case fair enough, they need to be ad-hoc.

(potentially we could have a slightly clever fallback mechanism where the string can be overridden when needs to be different, or default/fallback to the lower case version. That would have the added benefit of highlighting these differences when they're there...but I think that may be overkill and this is totally fine 👍)